### ISSUE TITLE
Doesn't show any of the addon buttons

### ISSUE THREAD

**Issue by thefunkygibbon** [k5md/Steam-Bulk-Sell-webextension#14]
If i go to my inventory it just shows as normal. 
if i click on the plugin it tells me i should be browsing to https://steamcommunity.com/profiles/thefunkygibbon/inventory

but that uri does not work.  it should be https://steamcommunity.com/id/thefunkygibbon/inventory  , if i go to any of my friends profile pages it is always "id" not "profiles"   could that be why it doesn't work?

**Comment by k5md** [comment]
Hello, @thefunkygibbon ! Thanks for creating the issue! I've checked the extension and it loads normally (shows the interface), however the api for item prices and sell requests has changed again, so no price checking and item selling is possible until I check steam's new improvements and release the patch fixing this tommorow (so really thanks for notification, since I dont bulk sell items often).

As for the problem: the numbers in link refer to steam community id (which is unique and cannot be changed like profile id) which can be seen in Edit profile -> General -> Custom URL when the field is empty (it's written here: https://help.steampowered.com/en/faqs/view/2816-BE67-5B69-0FEC ).
![issue14_0](https://github.com/user-attachments/assets/529ce6ad-5368-4930-91d6-109860a3bee5)

Or more conveniently by checking user's profile at any steam rep site as mentioned here: https://www.reddit.com/r/Steam/comments/gcowwz/comment/fpdfpxy/ . Hope this helps! Please, don't post any personal info in issue text, I've removed links to your profile.

**Comment by thefunkygibbon** [comment]
yeah i get that there is user id which is a base64 number and also normal id names.   most people these days use the username in the url rather than the id.  and clicking on the inventory within my own (and presumably it happens for most people these days) goes to /id/username  rather than profile/741243124124 or whatnot.    so is that what is causing it?  does your plugin only look and activate upon going to /profile/7741471471247 rather than just  looking for steamcommunity.com........./inventory?

fwiw if i do go to /profiles/741492786124 then it just redirects to /id/username

i've just moved over to firefox from years of using chrome and having a plugin which does this without issue. so i'm a little confused as to how this one is supposed to work.  totally granted i could be doing something wrong, but i'm trying to follow the instructions.

**Comment by k5md** [comment]
Lol, I've set custom URL and got your issue reproduced with steam redirecting me from /profiles page to the /id one, but not for every link and after removing custom URL I can't set any custom URL again...
![image](https://github.com/user-attachments/assets/0a062ea2-db4b-4b14-a5fa-119cf9695426)
Classic steam, almost as disgusting as polluting window object or the code in economy_v2.js or badges.js . However, I see, it's really necessary to add "*://steamcommunity.com/id/*/inventory*" to manifest's content_script matches and that's the only change needed, since steam does not use normal id names anywhere, only base64 number, so thanks, I'll release the update in the morning.
>i've just moved over to firefox from years of using chrome and having a plugin which does this without issue
That's great, I'm interested in looking through its sources!

**Comment by thefunkygibbon** [comment]
thanks, I see its working now.  but can i suggest some improvements (to bring it in line with the chrome one)
well, there are a few, but the main one really that i can see from just briefly trying it, is to be able to select cards by clicking one card and then holding shift and clicking the last card and it selects all of the cards between them.     clicking manually on each card/item when you may have multiple pages of them is a big ol faff still.
